### PR TITLE
Wei

### PR DESCRIPTION
**Pull request by songwei1990**
added test suites for Bob's pull request

### CODE REVIEW

File: src/test/java/origamiV2Tests/AppHeaderTest.java [hunk 1]
```diff
@@ -357,7 +357,7 @@ private void userMenuForBasicModeTest() throws Exception {
         };
     }
 
-    @Test(testName = "BasicMode - Add courses", dataProvider = "BasicMode-Add Course", groups = {"desktop-regression", "origamiV2"}, priority = 1)
+    @Test(testName = "BasicMode - Add courses", dataProvider = "BasicMode-Add Course", groups = {"desktop-regression", "origamiV2"})
```
Comment (umahaea) [inline src/test/java/origamiV2Tests/AppHeaderTest.java]: Why do we have to make any change in app-header test for it? Can we revert to original?

File: pom.xml [hunk 2]
```diff
@@ -43,8 +43,9 @@
                 <version>2.5</version>
                 <configuration>
                     <testFailureIgnore>false</testFailureIgnore>
-                    <suiteXmlFiles> 
-                        <suiteXmlFile>test_suites/${test_suite_xml}</suiteXmlFile>
+                    <suiteXmlFiles>
+
+                        <suiteXmlFile>test_suites/contextual_help.xml</suiteXmlFile>
```
Comment (umahaea) [inline pom.xml]: @songwei1990  - pls set this back to  <suiteXmlFile>test_suites/${test_suite_xml}</suiteXmlFile>

File: src/test/java/origamiV2Tests/ContextualHelpTest.java [hunk 3]
```diff
@@ -299,6 +299,32 @@ private void openSpecificHelpTopicTest() throws Exception {
             Assert.assertTrue(result);
         }
     }
+    
+	@Test(testName = "Verify that setLanguage method will not make contextual detail visible", groups = {
```
Comment (umahaea) [inline src/test/java/origamiV2Tests/ContextualHelpTest.java]: @songwei1990  - Mobile test is missing for this change. We need to add one.

File: src/test/java/origamiV2Tests/AppHeaderTest.java (hunk 1 above)
Comment (songwei1990) [inline src/test/java/origamiV2Tests/AppHeaderTest.java]: sure! sorry for this

**Pull request merged**